source/support.tex

```diff
@@ -837,6 +856,11 @@
 Freestanding implementations should only define a macro from \libheader{version}
 if the implementation provides the corresponding facility in its entirety.
 
+\pnum
+\recommended
```

Comment:
Does it make sense to have 2 \recommendeds following each other like this, verses having a bulleted list under a single \recommended?

Reply:
I think it would be stylistically better if there was a single paragraph, and I don't even think that bullets are required. It's not really wrong, and CWG signed off on it knowing that there are two recommendations in a row, so I'm not sure if that's a fixup candidate.